client/lib/polyfills.js

```diff
@@ -88,9 +88,41 @@ function fillPolyfill () {
   }
 }
 
+function findIndexPolyfill () {
+  if (!Array.prototype.findIndex) {
+    Object.defineProperty(Array.prototype, 'findIndex', {
+      value: function(predicate) {
+        'use strict';
+        if (this == null) {
+          throw new TypeError('Array.prototype.findIndex called on null or undefined');
+        }
+        if (typeof predicate !== 'function') {
+          throw new TypeError('predicate must be a function');
+        }
+        var list = Object(this);
+        var length = list.length >>> 0;
```

Comment:
I'm not sure I fully understand how the length value is build with the operator ">>>" From the [doc](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Operators/Bitwise_Operators#Unsigned_right_shift) I understand that it will never a negative value, is it use for that on this case?